editor/components/post-publish-panel/postpublish.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress Dependencies
+ */
+import { PanelBody, Button, ClipboardButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal Dependencies
+ */
+import { getCurrentPost } from '../../store/selectors';
+
+class PostPublishPanelPostpublish extends Component {
```

Comment:
A bit of inconsistency here between the name of the class and the name of the file. I'm not entirely sure if the correct spelling would be "Postpublish" (file name `postpublish.js`) or "PostPublish" (file name `post-publish.js`) but the two should align.

Reply:
Actually, there's no inconsistency. The `PostPublishPanel` in the classname refers to the foldeer and the `Postpublish` to the filename. Hard to distringuish between `PostPublish` and `Postpublish` though? Maybe we'd want to change the name for the component entirely?